subprojects/build-cache/src/integTest/groovy/org/gradle/caching/configuration/BuildCacheCompositeConfigurationIntegrationTest.groovy

```diff
@@ -96,7 +99,7 @@ class BuildCacheCompositeConfigurationIntegrationTest extends AbstractIntegratio
 
         and:
         def finalizeOps = operations.all(FinalizeBuildCacheConfigurationBuildOperationType)
-        finalizeOps.size() == 2
+        finalizeOps.size() == 5
```

Comment:
This is a break for build scans. 

We either need to roll this back (e.g. don't fire this event for non root builds), or we have to update build scans to ignore these events.